snapshot_tests/view/column_view_padding_static_children.scene.json

```diff
@@ -0,0 +1,49 @@
+{
+  "video": {
+    "root": {
+      "type": "view",
+      "background_color": "#000000FF",
+      "children": [
+        {
+          "type": "view",
+          "background_color": "#FF0000FF",
+          "direction": "column",
+          "width": 300,
+          "height": 300,
+          "children": [
+            {
+              "type": "view",
+              "padding_top": 10,
```

Comment:
you are not testing anything here, without any children padding does not do anything